app/controllers/api/v1/users_controller.rb

```diff
@@ -62,7 +63,12 @@ def user_params
 			end
 
 			def set_user
-				@user = User.find( id: params[ :id ] )
+				@user = User.find( params[ :id ] )
+				ExceptionHandler.record_not_found if @user.blank?
```

Comment:
If you chose #find it's better to catch RecoedNotFound exception at the controller method.